packages/visualizations/src/components/Table/types.ts

```diff
@@ -30,7 +34,10 @@ export type NumberColumn = BaseColumn & {
      *
      * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#options
      */
-    options?: Intl.NumberFormatOptions;
+    options?: {
+        display?: (v: string) => string;
+        intl?: Intl.NumberFormatOptions;
```

Comment:
We might want to explain with a comment how this two props works if both defined
1. Intl is done
2. display function run after